doc/scenarioParametersConfiguration.md

```diff
@@ -303,6 +299,53 @@ parameters:
       canChangeRowsNumber: false
       dateFormat: 'dd/MM/yyyy'
 ```
+#### Data source
+By default, table component is empty but is you want to display some data, you can define either a default or dynamic
```

Comment:
- `table component` -> `the table component` ? (I'm not sure if the first one is valid, if it is, then you can ignore this comment :smile:)
- `but is you want` -> `but if you want`